### PR TITLE
XSOAR CI/CD Article

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Status
In Progress

## Related Issues
fixes: https://github.com/demisto/etc/issues/42264
fixes: https://github.com/demisto/etc/issues/44792

## Description
XSOAR CI/CD Article.

### CODE REVIEW

**Review by guykeller — COMMENTED** [review]
@ChanochShayner As we talked privately. LMK when it is ready for a re-review

**Comment by ChanochShayner** [comment]
> @ChanochShayner As we talked privately. LMK when it is ready for a re-review

Ready for re-review

File: content-repo/extra-docs/articles/CI-CD-Article.md [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+Ensure that the server configuration (Settings>ABOUT>Troubleshooting)  `content.pack.verify` is set to `false` (enables you to import custom Content Packs that are not provided by Cortex XSOAR). 
+
+7. **(Artifact Server) Add the content to Cortex XSOAR.**
+   You can either use the [XSOAR CI/CD Content Pack](https://xsoar.pan.dev/docs/reference/packs/content-management) or download content manually. If using Google Cloud Services, download the [Google Cloud Storage Content Pack](https://xsoar.pan.dev/docs/reference/integrations/google-cloud-storage) and set up the integration instance. If using another storage application you need to download the appropriate Content Pack such as AWS. 
+   - Download the [XSOAR CI/CD Content Pack](https://xsoar.pan.dev/docs/reference/packs/content-management).
+   The Content Pack includes the Configuration Setup playbook, the configuration setup layout, incident fields,  automations, etc. The playbook runs via a job every 3 hours. The playbook fetches the configuration file and loads the contents to the machine. It downloads, and installs the custom content packs and configures lists and jobs if part of the content packs.
```
Comment (ChanochShayner) [inline content-repo/extra-docs/articles/CI-CD-Article.md:404]:
```suggestion
   The Content Pack includes the Configuration Setup playbook, the configuration setup layout, incident fields,  automations, etc. The playbook can run via a job or manually. The playbook fetches the configuration file and loads the contents to the machine. It downloads, and installs the custom content packs and configures lists and jobs if part of the content packs.
```

File: content-repo/extra-docs/articles/CI-CD-Article.md [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+    │   │   │   ├── 1.0.2
+    │   │   │   │   │   ├── pack2.zip
+    │   │   │   ├── 
+    │   │   ├──
+
+ ````
+
+6. **(Non-artifact server) Install the content to Cortex XSOAR**.
+<br/> In Cortex XSOAR go to Marketplace and install the custom content packs. 
+Ensure that the server configuration (Settings>ABOUT>Troubleshooting)  `content.pack.verify` is set to `false` (enables you to import custom Content Packs that are not provided by Cortex XSOAR).
```
Comment (ChanochShayner) [inline content-repo/extra-docs/articles/CI-CD-Article.md:399]: 1. In the non-artifacts-server option after the upload the packs already appear in the Installed Packs in the marketplace.
2. We don't need to configure the key `content.pack.verify` - its append automatically when using the demisto SDK Upload command.

File: content-repo/extra-docs/articles/CI-CD-Article.md [hunk 3]
```diff
@@ -0,0 +1,436 @@
...
+ ### Artifact Server
+
+An artifact server such as Google Cloud Services, enables you to  maintain and control version deployment and rollbacks, so you can keep track of your work. Each content pack version is saved in separate folders in the artifact server. You can use the [XSOAR CI/CD](https://xsoar.pan.dev/docs/reference/packs/content-management) Content Pack, to fetch the content from the artifact server to the Cortex XSOAR server. The content pack enables you to monitor, install content packs via a Playbook, incident type, etc. 
+
+
+
+![artifacts_server.png](../../../docs/doc_imgs/reference/XSOAR-CICD/artifacts_server.png)
+
+ **NOTE**: For Multi-tenant you cannot run the playbook in the XSOAR CI/CD Content Pack, as its not possible to create incidents or playbooks on the Main account. You can do one of the following:
+ - Upload the content via the CI/CD Build (it may still support the version control if you store all the versions locally. If using an artifact server, contact Customer support to upload via URL).
```
Comment (ChanochShayner) [inline content-repo/extra-docs/articles/CI-CD-Article.md:202]:
```suggestion
 - Upload the content via the CI/CD Build (Ref tp the relevant paragraph).
```

File: content-repo/extra-docs/articles/CI-CD-Article.md [hunk 4]
```diff
@@ -0,0 +1,436 @@
...
+- [Depoloyment](#deployment)
+
+For general information about the CI/CD process, see [CI/CD FAQs](#cicd-faqs).
+
+
+ ### Development
+ 
+ In the development stage, [set up the CI/CD process](#setup-the-cicd-development-process) by creating or cloning a Git repository based on the [CI/CD template repository](https://github.com/demisto/content-ci-cd-template). You can then create and maintain your custom content with the built-in functionality of [demisto-sdk](https://xsoar.pan.dev/docs/concepts/demisto-sdk). The CI/CD process uses `demisto-sdk` to develop and deploy custom content packs. It downloads, uploads, validates, creates content, migrates content from a Cortex XSOAR server, etc. It also supports content pack structure generation, auto validation of custom content including running linters on the code, and generation of content pack documentation, etc. For a full list of features, commands, and arguments, see [demisto-sdk-commands](https://github.com/demisto/demisto-sdk#commands).
+
+ **NOTE**: Playbook development is undertaken on the Cortex XSOAR server, which you can download to your machine.
```
Comment (ChanochShayner) [inline content-repo/extra-docs/articles/CI-CD-Article.md:38]: Please remove.

File: content-repo/extra-docs/articles/CI-CD-Article.md [hunk 5]
```diff
@@ -0,0 +1,436 @@
...
+    |`custom_packs`| Custom content packs, which are installed through the build process.|
+    | `marketplace_packs` |Marketplace packs to be installed on the machine.|
+    | `lists`| Lists that are created in the machine.|
+    | `jobs` | Jobs that are created in the machine.|
+
+    3.1 **Update the ID and URLs for each custom content pack.**
+    
+    For a non-artifact server, change the `URL` to the name of the Pack in the following format (using a local URL):
+     `"Url": "Packs\name of the pack\.zip”` 
+     <br/> **NOTE:** If you want to change the version you need to change it in the branch repository and not in this file.
```
Comment (ChanochShayner) [inline content-repo/extra-docs/articles/CI-CD-Article.md:285]: Please elaborate that this is refer to the non-artifacts - version control.

**Review by ChanochShayner — COMMENTED** [review]
Looks good!
Please see my comments.

**Comment by bakatzir** [comment]
fixes: https://github.com/demisto/etc/issues/42530

**Review by guykeller — APPROVED** [review]
Looks much better!

**Pull request merged**